BUILD_GDAL_35.md

```diff
@@ -0,0 +1,47 @@
+## Installing GDAL 3.5 from Cmake on Ubuntu
+
+Ideally, you can just install libgdal-dev from [ubuntugis unstable](https://launchpad.net/~ubuntugis/+archive/ubuntu/ubuntugis-unstable) but it has not been updated to 3.5 yet, so here are instructions to build from source using a conda environment on Ubuntu 20.04
+
+### Prerequisites
+
+Install g++11 and create a conda env
+
+```bash
+sudo add-apt-repository -y ppa:ubuntu-toolchain-r/test
+sudo apt install -y g++-11
```

Comment:
I suspect we actually need more than just g++-11 here and any reasonably modern ubuntu distro will be tracking gcc-11 or later. Did you actually run into issues with the default `g++` package? If not, I'd recommend we encourage people to do `sudo apt-get install build-essential` instead, since that will come with compilers, ensure gnu make is installed etc.